MonstieWash/Assets/Scripts/MonsterAi/MonsterBrain.cs

```diff
@@ -4,6 +4,7 @@
 using TMPro;
 using System;
 using UnityEngine.SceneManagement;
+using Unity.VisualScripting.Dependencies.NCalc;
```

Comment:
uhh not sure what the heck this is but it looks like it's not being used so let's get rid of it

```suggestion

```